google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -694,7 +694,13 @@ void WaitForConsistencyCheck(
     fut.get();  // simplify example by blocking until operation is done.
   }
   //! [wait for consistency check]
-  (std::move(old_admin), argv.at(2));
+  (old_admin, argv.at(2));
+
+  // TODO(#7740) - remove this extra call when lifetime issues are fixed.
+  //
+  // Use old_admin outside of the lambda to extend its lifetime long enough to
+  // avoid a crash.
+  (void)old_admin.ListTables(google::bigtable::admin::v2::Table::NAME_ONLY);
```

Comment:
Do we need to make a call, or just not `move` it in the parameter list?

Reply:
It still crashes without the `move` :/

Reply:
Perhaps the `ListTables()` is lasting long enough that the extended lifetime of `old_admin` avoids the problem?  Maybe a sleep would suffice too.

For what it's worth ...
```
Step #3: libc++abi: terminating
Step #3: *** SIGABRT received at time=1639503165 on cpu 4 ***
Step #3: PC: @     0x7fa5f8fc92a2  (unknown)  raise
Step #3:     @     0x7fa5fde468d0        112  absl::lts_20211102::WriteFailureInfo()
Step #3:     @     0x7fa5fde45ef7        112  absl::lts_20211102::AbslFailureSignalHandler()
Step #3:     @           0x481727        384  SignalAction()
```
implies to me that "libc++abi" called `abort(3)`.  It would be nice if it told us more than "terminating".

You said you can reproduce it ~2% of the time, yes?  Is so, perhaps one of asan/msan/tsan might shed some light.